frontend/bank-transfer-ui/src/app/main/transfer-list/transfer-list.component.html

```diff
@@ -1,10 +1,96 @@
-
-<div class="content">
-
+<div class="wrapper">
   <div class="container">
+    <div class="content">
 
-    <h1>Sample text</h1>
+      <h3 class="display-4 mb-3">Transfer history</h3>
+      <hr class="mb-3">
+      <div class="my-3">
+        <button type="button" class="btn btn-secondary btn-custom btn-lg" (click)="open(content)">Filter options
+        </button>
+      </div>
+      <div *ngFor="let tran of transfer | groupBy:'createTime'">
+        <p class="mb-1 mt-4">{{tran?.key | date:'fullDate'}}</p>
+        <ul class="list-group">
+          <li class="list-group-item list-group-item-dark list-group-item-custom" *ngFor="let list of tran.value">
+            <div class="row">
+              <div class="col-md-2 col-sm-3">{{list?.status}}</div>
+              <div class="col-md-3 col-sm-3">{{list?.title}}</div>
+              <div class="col-md-4 col-sm-auto">{{list?.accountNo}}</div>
+              <div class="col-md-2 col-sm-auto">{{list?.value | currency:'PLN':'code':'1.2-2':'pl'}}</div>
+            </div>
+          </li>
+        </ul>
+      </div>
 
+      <div class="search-results"
+           infiniteScroll
+           [infiniteScrollDistance]="85"
+           [infiniteScrollThrottle]="1300"
+           (scrolled)="onScroll()">
+      </div>
+      <ngx-spinner
+        bdColor="transparent"
+        size="medium"
+        color="rgb(113, 200, 55)"
+        loadingText=""
+        type="ball-clip-rotate">
+      </ngx-spinner>
+    </div>
   </div>
-
 </div>
+
+<ng-template #content let-c="close" let-d="dismiss">
+  <div class="modal-header">
+    <h4 class="modal-title" id="modal-basic-title">Filter options</h4>
+    <button type="button" class="close" aria-label="Close" (click)="d('Cross click')">
+      <span aria-hidden="true">&times;</span>
+    </button>
+  </div>
+  <div class="modal-body my-3">
```

Comment:
Jeśli każdy modal-body miałby mieć my-3, to warto rozważyć nadpisanie globalnych styli, a nie w każdym modalu dodawać my-3. Ewentualnie swój wrapper na okno modalne